e2e/vm/vm_test.go

```diff
@@ -82,3 +84,16 @@ var resetVM = func(o *option.Option, installed bool) string {
 
 	return limaConfigFilePath
 }
+
+var resetDisks = func(o *option.Option, installed bool) {
+	if installed {
+		path, err := exec.LookPath(e2e.InstalledTestSubject)
+		gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+		realFinchPath, err := filepath.EvalSymlinks(path)
+		gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+		gomega.Expect(os.RemoveAll(filepath.Join(realFinchPath, "../../lima/data/_disks")))
```

Comment:
we should delete the underlying disk in this case too

Reply:
updated in latest rev